pkg/manifests/nmstateconfig.go

```diff
@@ -0,0 +1,71 @@
+package manifests
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+
+	aiv1beta1 "github.com/openshift/assisted-service/api/v1beta1"
+	"github.com/openshift/assisted-service/models"
+	"github.com/pkg/errors"
+	"sigs.k8s.io/yaml"
+)
+
+func getNMStateConfig() aiv1beta1.NMStateConfig {
+	nmStateConfigData, err := os.ReadFile("./manifests/nmstateconfig.yaml")
+	if err != nil {
+		fmt.Errorf("Error reading pull secret: %w", err)
```

Comment:
Error message should refer to networkstateConfig (note, I've changed to not use these hardcoded messages in https://github.com/openshift-agent-team/fleeting/pull/15)